site

bernicelau430 opened the pull request.

bernicelau430 committed:
site

bernicelau430 committed:
removed unused images

neverabsolute commented:
i might be wrong but can't u import everything bootstrap with a 1-liner in the html?

bernicelau430 committed:
removed and simplified link imports

bernicelau430 commented:
Hey Jackson!

I just went back and replaced as many lines as I could with 1-liners so now
it’s just 15k. I added the changes to the existing PR. The files that I
contributed to are index-4.html and css/style.css so it would be great if
you could look over those. The rest of the code should just be from the
original template.

Thanks for your time and help!


- Bernice

On Mon, Feb 28, 2022 at 6:05 PM Jackson ***@***.***> wrote:

> im saying this because a 32000 PR is hard as shit to review, would need to
> know the specific filenames that aren't bootstrap files
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/zikegcwk/aicamp_emotion/pull/46#issuecomment-1054905489>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKLRQFDCNZSL5KHJLL7ROETU5QSWJANCNFSM5PQHHQ5Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
Best,
Bernice Lau
*Student, Class of '22*

Lowell High School
Tel: (415) 919-8023
Email: ***@***.***
LinkedIn: https://www.linkedin.com/in/lau-bernice/
Website: https://bernicelau.weebly.com/

neverabsolute reviewed (changes requested):
perhaps update the bottom of the readme? might not be 100% necessary but would be nice to cover basic structure and technologies used

bernicelau430 committed:
Update README.md

bernicelau430 commented:
Done! Updated the readme file. Let me know if you have any other
suggestions!

On Thu, Mar 3, 2022 at 5:38 PM Jackson ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> perhaps update the bottom of the readme? might not be 100% necessary but
> would be nice to cover basic structure and technologies used
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/zikegcwk/aicamp_emotion/pull/46#pullrequestreview-899722490>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKLRQFA3LX6KYCBRCC6JAR3U6FSQBANCNFSM5PQHHQ5Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
Best,
Bernice Lau
*Student, Class of '22*

Lowell High School
Tel: (415) 919-8023
Email: ***@***.***
LinkedIn: https://www.linkedin.com/in/lau-bernice/
Website: https://bernicelau.weebly.com/
